Polyhedron/demo/Polyhedron/Scene_c3t3_item.h

```diff
@@ -39,6 +39,28 @@ using namespace CGAL::Three;
   Scene_c3t3_item();
   Scene_c3t3_item(const C3t3& c3t3);
   ~Scene_c3t3_item();
+
+  enum STATS {
+    MIN_EDGES_LENGTH = 0,
+    MAX_EDGES_LENGTH,
+    MEAN_EDGES_LENGTH,
+    MIN_DIHEDRAL_ANGLE,
+    MAX_DIHEDRAL_ANGLE,
+    MEAN_DIHEDRAL_ANGLE,
+    NB_SPHERES,
+    NB_CNC,
+    NB_VERTICES,
+    NB_TETS,
+    SMALLEST_RAD_RAD,
+    SMALLEST_EDGE_RAD,
+    BIGGEST_VL3_CUBE,
+    NB_SUBDOMAINS
+  };
```

Comment:
That enum is part of the implementation internals of the item. It should be in the `.cpp` and not in the `.h`, to avoid a recompilation of all dependent code if that list changes.